pages/your_tasks/virtual_dev_environments.md

```diff
@@ -91,10 +90,24 @@ a [bad example of a spaghetti setup][python-env-hell], not knowing which depende
 
 * Decide on and start using a package manager tool and a virtual environment management tool for your programming language.
 
+### Overview of (some) virtual environment tools
+
+| Programming language | Virtual environment tools |
+| --- | --- |
+| Python | {% tool "venv" %}, {% tool "poetry" %}, {% tool "conda" %} |
+| R | {% tool "conda" %}, {% tool "renv" %} |
+| Julia | {% tool "pkg-jl" %} |
```

Comment:
```suggestion
| Python | {%- tool "venv" -%}, {%- tool "poetry" -%}, {%- tool "conda" -%} |
| R | {%- tool "conda" -%}, {%- tool "renv" -%} |
| Julia | {%- tool "pkg-jl" -%} |
```

Currently, the rendering is messed up in the preview. I am not sure if this will fix it, but it may help https://shopify.github.io/liquid/basics/whitespace/

Presently, it appears that the expansion order is `liquid` then `markdown`. So worst-case, we have to write this as inline html.